microprofile/WFLY-15006_reactive_messaging_with_rhosak.adoc

```diff
@@ -0,0 +1,167 @@
+= MicroProfile Reactive Messaging Integration with RHOSAK
+:author:            Kabir Khan
+:email:             kkhan@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+RHOSAK (Red Hat Openshift Streams for Apache Kafka) is a new Managed Service provided by Red Hat. It is important that Red Hat products that can run on OpenShift and support Kafka are able to use RHOSAK.
+
+The integration of MicroProfile Reactive Messaging introduced for MicroProfile XP 3 uses SmallRye Reactive Messaging as its implementation. This already works with RHOSAK from Quarkus.
+
+Initial manual testing (tweaking of the Reactive Messaging Quickstart) shows that it is possible to do the same with WildFly once patched with https://issues.redhat.com/browse/WFLY-14613[WFLY-14613].
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-15006[WFLY-15006]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-13640[WFLY-13640 - Original RFE bringing in MP Reactive Messaging + Kafka support]
+* https://issues.redhat.com/browse/WFLY-14987[WFLY-14987 - The MP Reactive Messaging Kafka connector should be able to use an SSLContext configured in the Elytron subsystem]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+* TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [x] QE
+
+=== Affected Projects or Components
+WildFly - minor changes to the module structure might be needed (such as https://issues.redhat.com/browse/JBEAP-21331[JBEAP-21331]) if further issues are found during testing. Apart from that this is largely expected to be a verification task.
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
```

Comment:
Could you please uncheck this installation type since this feature will be part of XP only in the product and as such won't be tested properly.